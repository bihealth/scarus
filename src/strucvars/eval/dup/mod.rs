//! Evaluation of duplications, gain of copy number.

pub mod result;
