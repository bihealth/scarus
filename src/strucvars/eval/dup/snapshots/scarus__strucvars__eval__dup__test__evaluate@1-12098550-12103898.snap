---
source: src/strucvars/eval/dup/mod.rs
expression: result
---
- G1:
    G1A:
      genes: []
      functional_elements:
        - RefSeq:
            chromosome: "1"
            start: 12098306
            stop: 12099428
            id: "id-GeneID:115801416"
            dbxref: "GeneID:115801416"
            category: 6
            regulatory_class: 16
            note: ~
            experiment: "EXISTENCE:mutant phenotype evidence [ECO:0000015][PMID:30612741]"
            function: positively regulates MIIP gene expression in K562 cells
        - RefSeq:
            chromosome: "1"
            start: 12098306
            stop: 12099428
            id: "id-GeneID:115801416-2"
            dbxref: "GeneID:115801416"
            category: 5
            regulatory_class: ~
            note: ~
            experiment: ~
            function: "regulatory_interactions: MIIP"
- G3:
    G3A:
      suggested_score: 0
      num_genes: 0
      genes: []

